pipeline/backend/kubernetes/pod.go

```diff
@@ -182,7 +183,7 @@ func podContainer(step *types.Step, podName, goos string, options BackendOptions
 	container := v1.Container{
 		Name:            podName,
 		Image:           step.Image,
-		WorkingDir:      step.WorkingDir,
+		WorkingDir:      step.WorkSpaceBase,
```

Comment:
Everywhere else in the code, this var is named `WorkspaceBase` can we name keep this naming?